.github/workflows/ci.yaml

```diff
@@ -43,6 +43,43 @@ jobs:
         run: |
           pytest --durations-min=5 --ignore=src/tests/integration
 
+  test-slalchemy-v1:
```

Comment:
```suggestion
  test-sqlalchemy-v1:
```

Reply:
thanks! I've remember using nox in the past but encountered problems with conda; perhaps it's time to give it a try again!